sympy/discrete/transforms.py

```diff
@@ -63,14 +65,21 @@ def _fourier_transform(seq, dps, inverse=False):
                 a[i + j], a[i + j + hf] = u + v, u - v
         h *= 2
 
+    n = sympify(n)
     if inverse:
-        a = [(x/n).evalf(dps) for x in a] if dps is not None \
-                            else [x/n for x in a]
+        denom = n**((1+alpha)/2)
+    else:
+        denom = n**((1-alpha)/2)
```

Comment:
Maybe just `denom = n**((1 + alpha)/S(2)) if inverse else n**((1 - alpha)/S(2))` instead of lines 68-72.